apps/api-server/src/routes/api/resource.js

```diff
@@ -91,6 +95,12 @@ router.all('*', function (req, res, next) {
     req.scope.push('includeTags');
```

Comment:
includeTags zou eigenlijk ook wel naar mijn idee standaard als scope aan staan. Het is niet veel code en zal ook niet voor een grote vertraging zorgen naar mijn idee. Daarbij elke keer als we een resource opvragen dan zetten we daar includeTags bij, dus dit kan volgens mij gewoon een default scope worden

Reply:
Ik denk dat je gelijk hebt dat het vaak meegstuurd zal worden, maar om het er dan maar standaard aan te hangen gaat mij wat te ver. Een smaak kwestie denk ik, maar imo moet een api zich daarin wat terughoudend gedragen, en geen mening hebben over hoe data gebruikt zal gaan worden.

Voor statussen is dat anders: die zijn, ook binnen de api, functioneel noodzakelijk.